mxnet/inference/docker/1.9/py3/Dockerfile.graviton

```diff
@@ -0,0 +1,132 @@
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG MMS_VERSION=1.1.2
+ARG MX_URL=https://repo.mxnet.io/dist/python/cpu/mxnet-1.9.0b20210914-py3-none-manylinux2014_aarch64.whl
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.8.10
+ARG OPENSSL_VERSION=1.1.1l
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib" \
+    PYTHONIOENCODING=UTF-8 \
+    LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    TEMP=/home/model-server/tmp
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    cmake \
+    libprotobuf-dev \
+    protobuf-compiler \
+    build-essential \
+    ca-certificates \
+    curl \
+    emacs \
+    git \
+    libopencv-dev \
+    openjdk-8-jdk-headless \
+    openjdk-8-jdk \
+    openjdk-8-jre \
+    vim \
+    wget \
+    unzip \
+    zlib1g-dev \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    libgdbm-dev \
+    libc6-dev \
+    libbz2-dev \
+    tk-dev \
+    libffi-dev \
+    python3-h5py \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# install OpenSSL
+RUN wget -q -c https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz \
+ && tar -xzf openssl-${OPENSSL_VERSION}.tar.gz \
+ && cd openssl-${OPENSSL_VERSION} \
+ && ./config && make -j $(nproc) && make install \
+ && ldconfig \
+ && cd .. && rm -rf openssl-* \
+ && rmdir /usr/local/ssl/certs \
+ && ln -s /etc/ssl/certs /usr/local/ssl/certs
+
+# install ArmPL
+run wget https://developer.arm.com/-/media/Files/downloads/hpc/arm-performance-libraries/21-1-0/Ubuntu20.04/arm-performance-libraries_21.1_Ubuntu-20.04_gcc-9.3.tar \
+ && tar -xvf arm-performance-libraries_21.1_Ubuntu-20.04_gcc-9.3.tar \
+ && arm-performance-libraries_21.1_Ubuntu-20.04_gcc-9.3/arm-performance-libraries_21.1_Ubuntu-20.04.sh -a \
+ && export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/arm/armpl_21.1_gcc-9.3/lib/
+ 
+# install Python
+RUN wget -q https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xzf Python-$PYTHON_VERSION.tgz \
+ && cd Python-$PYTHON_VERSION \
+ && ./configure --enable-shared --prefix=/usr/local \
+ && make -j $(nproc) && make install \
+ && cd .. && rm -rf ../Python-$PYTHON_VERSION* \
+ && ln -s /usr/local/bin/pip3 /usr/bin/pip \
+ && ln -s /usr/local/bin/$PYTHON /usr/local/bin/python \
+ && ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+WORKDIR /
+
+RUN ${PIP} install --no-cache-dir \
+    "awscli<2" \
+    boto3 \
+    ${MX_URL} \
```

Comment:
`numpy` is not installed explicitly as it is being installed as a part of other package dependency